devdocs/process.md

```diff
@@ -68,8 +66,9 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    to your branch
         
        ```
-       git pull upstream 
-       git merge master
+       git fetch upstream 
+       git checkout master
+       git merge upstream/master
```

Comment:
Tested.

Set-up steps:

0. Created a new repo (`REPO_A`) with a file, `a.txt`, committed.
0. Pushed `REPO_A` to remote.
0. Forked remote `REPO_A` to `REPO_B` on another account.
0. Pull remote `REPO_B` to local.
0. Created a new branch (`BRANCH`) on `REPO_B`.
0. Create file, `b.txt`, on `REPO_B` and commit while in `BRANCH`.
0. Create file, `c.txt`, on `REPO_A`, push to remote.

I first went with the initial instructions:

0. Pull from `REPO_A` to `REPO_B` while in `BRANCH`.

Here are where the files went (`local`) after those steps:
```
REPO_A:master
> a.txt, c.txt

REPO_B:master
> a.txt

REPO_B:BRANCH
> a.txt, b.txt, c.txt
```

Next, I continued with my suggested sequence:

0. Create file, `d.txt`, on `REPO_A` and commit.
0. Pushed `REPO_A` to remote.
0. Pull from `REPO_A` to `REPO_B` while in `master`.

```
REPO_A:master
> a.txt, c.txt, d.txt

REPO_B:master
> a.txt, c.txt, d.txt

REPO_B:BRANCH
> a.txt, b.txt, c.txt
```
And the second part:

0. Switched to `BRANCH` on `REPO_B`.
0. Merged `master` while in `BRANCH` (`git merge master`).

```
REPO_A:master
> a.txt, c.txt, d.txt

REPO_B:master
> a.txt, c.txt, d.txt

REPO_B:BRANCH
> a.txt, b.txt, c.txt, d.txt
```

It would depend on whether the contributor intends to keep the forked master updated, but I believe this should generally work better?